api/endpoints/community.py

```diff
@@ -27,6 +29,14 @@ def make_blueprint(con):
         "disability_rate_2019":lambda: metric.disability_rate(year=2019, segment="all"),
         "belonging_rate_2017": lambda: metric.belonging(year=2017, segment="all"),
         "belonging_rate_2018": lambda: metric.belonging(year=2018, segment="all"),
+        "rent_burdened_2015": lambda: metric_rbu.rent_burdened(year=2015, segment="all"),
+        "rent_burdened_2016": lambda: metric_rbu.rent_burdened(year=2016, segment="all"),
+        "rent_burdened_2017": lambda: metric_rbu.rent_burdened(year=2017, segment="all"),
```

Comment:
For now, let's just load the years 2017-2019, that should be plenty for this view.

Later if you want, you can add a timeline metric that shows the value over the years.